CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# CONTRIBUTING TO COOKIECUTTER
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given. 
+
+
+    .. toctree::
```

Comment:
toctree not working in markdown. Should be removed or made manually.

Reply:
I was verify and believe sphinx not provides a direct support to Markdown.
There is a way for this, using [recommonmark](https://recommonmark.readthedocs.io/en/latest/auto_structify.html), see [here](https://www.sphinx-doc.org/en/1.6/markdown.html)...

I am verifying it...